client/state/themes/selectors.js

```diff
@@ -513,6 +513,17 @@ export function hasActivatedTheme( state, siteId ) {
 }
 
 /**
+ * Whether the theme is currently being installed on the (Jetpack) site.
```

Comment:
I think I'll leave it in this case. As a dev, I might look exactly for a note of this kind, knowing that `siteId` could potentially be a WPCOM site.